### PR TITLE
support lowercase jira keys

### PR DESCRIPTION
**Pull request by miniscruff**
Fixes BERTE-550 and part of my sanity
( I had actually did this a while ago and forgot about it... )

Edit: Added 3.6.6 changelog as well

### CODE REVIEW

**Comment by miniscruff** [comment]
Should be ready to review, I can't create a new Jira version so maybe @tcarmet can fix this for me

**Review by bertranddemiddelaer — APPROVED** [review]
LGTM

File: CHANGELOG [hunk 1]
```diff
@@ -1,6 +1,14 @@
...
 All notable changes to this project will be documented in this file.
 
+## [3.6.7] - 2021-11-09
+# Changed
+- Allow jira issue keys to be lowercase
+
+## [3.6.6] - 2021-11-08
+# Changed
+- Track new pattern for epic branches
+
```
Comment (tcarmet) [inline CHANGELOG:11]: Thank you :heart:

Comment (miniscruff) [reply CHANGELOG:11]: Would of annoyed the hell out of me if I didn't include it...

**Pull request merged**